pkg/domain/infra/abi/manifest.go

```diff
@@ -189,7 +175,7 @@ func (ir *ImageEngine) remoteManifestInspect(ctx context.Context, name string, o
 		if err != nil {
 			return nil, fmt.Errorf("parsing manifest blob %q as a %q: %w", string(result), manType, err)
 		}
-		list, err := listBlob.ConvertToMIMEType(manifest.DockerV2ListMediaType)
+		list, err := listBlob.ConvertToMIMEType(manType)
```

Comment:
Isn’t this doing a conversion to the format the list is already in? That could just be dropped.

[It might be possible to drop the whole parse+`Serialize` round-trip, and just unmarshal the raw data from `GetManifest`. I… don’t know that that’s desirable; a malicious manifest can contain _both_ single-platform and multi-platform data, and that ambiguity should either be detected and rejected, or at least resolved in interpretation, at the point of input. So, all in all, I actually quite like the re-serialization step. Or, alternatively, `ManifestListData` could be populated manually from the data of `manifest.*From*`, but that’s tedious.]

Reply:
Removed all of that and still seems to work.

Reply:
I’d prefer to leave the `ListFromBlob`+`Serialize`, to canonicalize ambiguous data.